prov/usnic/src/usdf_cq.c

```diff
@@ -954,9 +716,9 @@ static struct fi_ops_cq usdf_cq_context_ops = {
 static struct fi_ops_cq usdf_cq_context_soft_ops = {
 	.size = sizeof(struct fi_ops_cq),
 	.read = usdf_cq_read_context_soft,
-	.readfrom = usdf_cq_readfrom_context_soft,
+	.readfrom = usdf_cq_readfrom_context,
```

Comment:
Yeah, I agree that this looks like it's using hard CQ logic for a soft CQ.  Probably better to just stick in a stub right now that returns `FI_ADDR_NOTAVAIL` and file an issue to remember to come back to it later.